TempoAgents/Source/TempoAgentsShared/Public/TempoIntersectionQueryComponent.h

```diff
@@ -52,6 +52,11 @@ class TEMPOAGENTSSHARED_API UTempoIntersectionQueryComponent : public UActorComp
 	
 	virtual bool TryGetMinLaneIndexInLaneConnections(const TArray<FTempoLaneConnectionInfo>& LaneConnectionInfos, int32& OutMinLaneIndex) const;
 	virtual bool TryGetMaxLaneIndexInLaneConnections(const TArray<FTempoLaneConnectionInfo>& LaneConnectionInfos, int32& OutMaxLaneIndex) const;
+
+	// These return the indices, relative to the start or end of the spline, of a connected road Actor's spline that
+	// correspond to the intersection entrance point when that road Actor's spline starts or ends at this intersection.
+	virtual int32 GetIntersectionEntranceStartIndex() const;
```

Comment:
I think we should name these more properly as `GetIntersectionEntranceStartOffsetIndex` and `GetIntersectionEntranceEndOffsetIndex` for clarity.

Of course, this means modifying the project-side "overrides" PR as well.